src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js

```diff
@@ -11,9 +11,9 @@ export default BaseAuthenticatorView.extend({
     const selectedMethod = currentAuthenticator?.methods[0];
     if (selectedMethod?.type === 'push') {
       this.Body = ChallengeOktaVerifyPushView;
-      this.Footer = AuthenticatorVerifyFooter;
     } else {
       this.Body = ChallengeOktaVerifyFastPassView;
     }
+    this.Footer = AuthenticatorVerifyFooter;
```

Comment:
We might want to implement like this https://github.com/okta/okta-signin-widget/pull/1554/files#diff-ce9cd727f6eece71587db25b878b349a5235dfe959e1d713085c6bcc070b4e86R17

Reply:
What's the reason?

Reply:
That's UX design. And this is what i meant for the comment above.